webpack.client.config.js

```diff
@@ -107,14 +108,21 @@ function createJsPo(srcPo, lang) {
    * possible terminal paths.
    */
   const scriptsDir = shellQuote.quote([dirs.SCRIPTS]);
-  const nestedDirs = shell.exec(`find ${scriptsDir} -type d`, {silent: true}).stdout.split('\n');
+  const nestedDirs = shell.exec(
+    `find ${scriptsDir} -type d`,
+    {silent: true}
+  ).stdout.split('\n');
   const msgLocations = nestedDirs
     .filter(Boolean)
-    .map(dir => '-N ' + shellQuote.quote(['..' + dir.replace(dirs.CHECKOUT, '') + '/*.js']))
+    .map(dir => (
+      '-N ' +
+      shellQuote.quote(['..' + dir.replace(dirs.CHECKOUT, '') + '/*.js'])))
```

Comment:
```suggestion
      shellQuote.quote(['..' + dir.replace(dirs.CHECKOUT, '') + '/*.js']))
    )
```

Reply:
This one gets hit with `Unexpected newline before ')'.eslintfunction-paren-newline
`